aries-site/src/examples/foundation/content-container-sizes/BorderWidths.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Box, FormField, Select } from 'grommet';
+import { useDesignTokens } from '../../../components/content/designTokenUtils';
+
+export const BorderWidths = () => {
+  const [width, setWidth] = React.useState('default');
+  const { data } = useDesignTokens('semantic.borderWidth');
+
+  const containerTokens = data.map(
+    token => token.token.split('hpe.borderWidth.')[1],
+  );
+
+  return (
+    <Box direction="row" justify="between">
+      <Box
+        pad={{ vertical: 'xlarge', horizontal: 'xlarge' }}
+        align="center"
+        justify="center"
```

Comment:
```suggestion

```
do we need justify?

Reply:
It's needed for the ContainerSizes example when a height chosen is quite small -- just a copy-paste of the example container -- so technically not needed for the BorderWidths.

Reply:
ah okay